app/shared/json_convertible.rb

```diff
@@ -10,7 +10,7 @@ def self.included(base)
 
     # add these as instance methods
     module InstanceMethods
-      def to_json(options)
+      def to_json(options = {})
```

Comment:
without this, i was getting the `Expected 1 arg but got 0` error

Reply:
Nice catch!